### ISSUE TITLE
Inconsistencies in paths between ICS

### ISSUE THREAD

**Issue by fedekunze** [cosmos/ibc#345]
The paths are inconsistent between ICS. For example:

- [ICS24](https://github.com/cosmos/ics/blob/master/spec/ics-024-host-requirements/README.md#path-space): `"clients/{identifier}/consensusState"`
- [ICS07](https://github.com/cosmos/ics/blob/master/spec/ics-007-tendermint-client/README.md) : `"consensusStates/{identifier}/{header.height}"`
- [ICS02](https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#path-space): `"clients/{id}/consensusState"`

**Comment by cwgoes** [comment]
Yes, they are. ICS 7 supersedes the others. Thanks for the issue, I'll fix these.

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
